core/scripts/i18n/collect-strings.js

```diff
@@ -730,8 +730,16 @@ function checkKnownFixedCollisions(strings) {
       'Potential Savings',
       'The page was evicted from the cache to allow another page to be cached.',
       'The page was evicted from the cache to allow another page to be cached.',
+      'Use $MARKDOWN_SNIPPET_0$ to detect unused JavaScript code. $LINK_START_0$Learn more$LINK_END_0$',
```

Comment:
a few of these look like it's a waste to translate twice. e.g.

https://github.com/GoogleChrome/lighthouse-stack-packs/blob/db09b8af572e9041f8da7ce1aa608467a43626b6/packs/gatsby.js#LL12C7-L12C147

and

https://github.com/GoogleChrome/lighthouse-stack-packs/blob/db09b8af572e9041f8da7ce1aa608467a43626b6/packs/next.js#LL12C7-L12C147

are already the same, just the gatsby one is missing the trailing `.`

Reply:
made a note for this here https://github.com/GoogleChrome/lighthouse-stack-packs/issues/94